custom/ilsgateway/slab/utils.py

```diff
@@ -13,12 +13,12 @@ def get_current_state(sql_location):
         stock_state.sql_product.code: stock_state.get_monthly_consumption()
         for stock_state in StockState.objects.filter(
             case_id=sql_location.supply_point_id
-        ).select_related('sql_product__code').order_by('sql_product__code')
+        ).select_related('sql_product').order_by('sql_product')
```

Comment:
Why did the `order_by` have to change? 5 lines down uses `sql_prodcut__code` to order